src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -3,7 +3,12 @@
 public class Division {
 
     public static final long division(int a, int b) {
-        long result = ((long) a) / ((long) b);
+       // Expection Handler
```

Comment:
Line 6 - 11 adds the Expection Handler for when the division is an extreme case (Ex: 0/0, 1/0, ...)

Reply:
Looking Good